src/s1reader/s1_reader.py

```diff
@@ -349,28 +379,47 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
     # burst sensing starts in prototyping test data
     burst_interval = 2.758277
 
-    #parse manifest.safe to retrieve IPF version
+    # parse manifest.safe to retrieve IPF version
     manifest_path = os.path.dirname(annotation_path).replace('annotation','') + 'manifest.safe'
     with open_method(manifest_path, 'r') as f_manifest:
         tree_manfest = ET.parse(f_manifest)
         ipf_version = get_ipf_version(tree_manfest)
 
-    #Load the Product annotation - for EAP calibration
-    #with open_method(annotation_path, 'r') as f_lads:
-    #    tree_lads = ET.parse(f_lads)
-    #    product_annotation = s1_annotation.ProductAnnotation.from_et(tree_lads)
+    # Load the Product annotation - for EAP calibration
+    with open_method(annotation_path, 'r') as f_lads:
+        tree_lads = ET.parse(f_lads)
+        product_annotation = ProductAnnotation.from_et(tree_lads)
 
-    #load the Calibraton annotation
-    calibration_annotation_path = annotation_path.replace('annotation/', 'annotation/calibration/calibration-')
+    # load the Calibraton annotation
+    calibration_annotation_path =\
+        annotation_path.replace('annotation/', 'annotation/calibration/calibration-')
     with open_method(calibration_annotation_path, 'r') as f_cads:
         tree_cads = ET.parse(f_cads)
-        calibration_annotation = s1_annotation.CalibrationAnnotation.from_et(tree_cads)
+        calibration_annotation =\
+            CalibrationAnnotation.from_et(tree_cads,
+                                          calibration_annotation_path)
 
-    #load the Noise annotation
+    # load the Noise annotation
     noise_annotation_path = annotation_path.replace('annotation/', 'annotation/calibration/noise-')
     with open_method(noise_annotation_path, 'r') as f_nads:
         tree_nads = ET.parse(f_nads)
-        noise_annotation = s1_annotation.NoiseAnnotation.from_et(tree_nads, ipf_version)
+        noise_annotation = NoiseAnnotation.from_et(tree_nads, ipf_version,
+                                                   noise_annotation_path)
+
+    # load AUX_CAL annotation
+    flag_apply_eap = is_eap_correction_necessary(ipf_version)
+    if flag_apply_eap.phase_correction:
+        path_aux_cals = f'{os.path.dirname(s1_annotation.__file__)}/data/aux_cal'
+        path_aux_cal_zip = get_path_aux_cal(path_aux_cals,
+                                            platform_id,
+                                            product_annotation.inst_config_id)
+
+        aux_cal_subswath = AuxCal.load_from_zip_file(path_aux_cal_zip,
+                                                     pol,
+                                                     subswath_id)
```

Comment:
Does it work if EAP is not in a zip file?

Reply:
No it will not work, and it is our intention to load AUX_CAL data from .zip file, rather than from .xml file because of the reasons described [here](https://github.com/opera-adt/s1-reader/discussions/61#discussioncomment-3647149).